WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -359,6 +358,42 @@ def _getSecurityCFG( self ):
       self.cfg.append( "-o /DIRAC/Security/CertFile=%s/hostcert.pem" % self.pp.certsLocation )
       self.cfg.append( "-o /DIRAC/Security/KeyFile=%s/hostkey.pem" % self.pp.certsLocation )
 
+class CheckCECapabilities( CommandBase ):
+  """ Used to get  CE tags.
+  """
+  def __init__( self, pilotParams ):
+    """ c'tor
+    """
+    super( CheckCECapabilities, self ).__init__( pilotParams )
+    self.cfg = []
+
+
+  def execute( self ):
+
+    retCode, resourceDict = self.executeAndGetOutput( 'dirac-resource-get-parameters -S %s -N %s -Q %s' %
```

Comment:
So, after test: we can't use gConfig here, and this is clear as soon as we do "from DIRAC import gConfig" or any other "from DIRAC* import *" in any part of the pilot code. 2 reasons:
- if (VO)DIRAC is installed with dirac-install, it will still complain that can't find GSI. And this is because the environment at this point is not set (it's not in the current process)
- if (VO)DIRAC is installed from, say, CVMFS, still the environment created has to be passed in to the subprocess that run the scripts.